src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -208,6 +208,27 @@ public static Category parseCategory (String category) throws ParseException {
         return new Category(trimmedCategory);
     }
 
+    /**
+     * Parses a {@code String highOrLow}
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @param highOrLow to sort by ascending or descending order.
+     * @return the String
+     */
+    public static String parsehighOrLow(String highOrLow) throws ParseException {
+        requireNonNull(highOrLow);
+        String trimmedHighOrLow = highOrLow.trim();
+        if (!(trimmedHighOrLow.equals("high") ^ trimmedHighOrLow.equals("low"))) {
+            throw new ParseException("String must consist of either high for descending order"
+                    + " or low for ascending order");
+        }
+        if (trimmedHighOrLow.equals("low")) {
+            return "low";
+        } else {
+            return "high";
+        }
```

Comment:
Suggestion to use switch statements and let the default case throw the error, just like the AB3 Parser. If you take my above suggestion, then you can do the following:
```suggestion
    switch(highOrLow) {
    case SortFixedExpenseCommand.SORT_HIGH:
        return SortFixedExpenseCommand.SORT_HIGH;
    case SortFixedExpenseCommand.SORT_LOW:
        return SortFixedExpenseCommand.SORT_LOW;
    default:
        throw new ParseException()
     }
```